src/main/java/org/roy/loadx/transaction/TransactionPrinter.java

```diff
@@ -81,6 +81,16 @@ private String getTransactionDisplayName(String transactionName) {
 		}
 	}
 	
+	static private void println() {
+		System.out.println();
+		System.out.flush();
+	}
+	
+	static private void println(Object obj) {
```

Comment:
PrinterStream methods are typed, better to pass as String to avoid potential behavior differences. API to client more restrictive.